scripts/custom_lint_checks.py

```diff
@@ -297,6 +306,15 @@ def check_functiondef_params(self, node, node_doc):
             node_doc, node.args, node, node_allow_no_param)
 
     def check_functiondef_returns(self, node, node_doc):
+        """Checks whether a function documented with a return value actually has
+            a return statement in its definition
```

Comment:
nit: remove indentation, and add full stop at the end.

Reply:
done